spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  before(:each) do
+    @oregon_facilities = DmvDataService.new.or_dmv_office_locations
+    @new_york_facilities = DmvDataService.new.ny_dmv_office_locations
+    @missouri_facilities = DmvDataService.new.mo_dmv_office_locations
+    @factory = FacilityFactory.new
+  end
+
+  describe '#initialize' do 
+    it 'exists' do 
+      expect(@factory).to be_a FacilityFactory
+    end
+
+    it 'begins without facilities of any type' do 
+      expect(@factory.or_facilities).to eq([])
+      expect(@factory.ny_facilities).to eq([])
+      expect(@factory.mo_facilities).to eq([])
```

Comment:
Once you check this for one factory instance, really no need to check for all the factory instances. Over testing.